package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -639,7 +639,10 @@ def _getMeanOnePoint(self,universe,selection1,selection_str,dt,totalFrames):
         valdipList = []
 
         for j in range(totalFrames/dt-1):
-            a = self._getOneDeltaPoint(universe,repInd,j,sumsdt,dt)
+            try:
+                a = self._getOneDeltaPoint(universe,repInd,j,sumsdt,dt)
+            except ZeroDivisionError:
+                continue
```

Comment:
If you are sure that this is the best fix then please add at least a comment to the code explaining under which conditions you expect the exception to occur, e.g., elaborate on "happens when the selection of atoms is too small".